chapters/annotations.tex

```diff
@@ -268,6 +268,12 @@ \section{Annotations for Simulation Experiments}\doublelabel{annotations-for-sim
 relative integration tolerance (Tolerance) for simulation experiments to
 be carried out with the model or block at hand. If StartTime is not specified it is assumed to be \lstinline!0.0!.
 
+The experiment annotation can be inherited, the local experiment (if existing) only override the specific options in the normal way for modifiers.
```

Comment:
I find _specific options_ a bit unclear.  It gets easier to explain by talking about inheritance of experiment options.  First like this (cannot add GitHub suggestion this far away):
```
The \lstinline!experiment!-annotation defines default experiment options for simulation experiments to be carried out with the model or block at hand.
The options are only defaults in the sense that tools generally allow the options to be overridden at simulation start.
The experiment options consist of \lstinline!StartTime! in {[}s{]} for the start time, \lstinline!StopTime! in {[}s{]} for the stop time, \lstinline!Interval! in {[}s{]} for a suitable time resolution of the result grid, and \lstinline!Tolerance! for the relative integration tolerance.
If \lstinline!StartTime! is not specified, the default start time is \lstinline!0.0!.
```
Then:
```suggestion
The experiment options are inherited, and the derived class may override individual inherited options.
```